test/openssl/test_bn.rb

```diff
@@ -330,6 +330,32 @@ def test_get_flags_and_set_flags
     e.set_flags(0)
     assert_equal(4, e.get_flags(OpenSSL::BN::CONSTTIME))
   end
+
+  def test_ractor
+    # Ractor support is from CRuby 3
+    return if Gem::Version.create('3.0') > Gem::Version.create(RUBY_VERSION)
+    return unless RUBY_ENGINE == 'ruby'
```

Comment:
Does `return unless defined?(Ractor)` work?

Reply:
I'm not sure which one to use, I'll use `defined?(Ractor)` 🙆‍♂️